README.CentOS7.md

```diff
@@ -36,6 +36,11 @@ sudo yum update
 sudo yum install php72
 sudo yum install php72-php-fpm php72-php-gd php72-php-json php72-php-mbstring php72-php-mysqlnd php72-php-xml php72-php-xmlrpc php72-php-opcache php72-php-pdo php72-php-mysql
 ```
+### PHP MySQL Packages
```

Comment:
same as for #5485, not the right fix.
#5489 will fix all 3 issues

Reply:
Hi @PapillonMcGill, I was getting these output errors after #5489. I think this PR and #5485 are actually required.

Reply:
@PapillonMcGill I'll try re-testing on a fresh install of CentOS and see what happens.

Reply:
@PapillonMcGill this PR is needed to resolve the PDO not found https://github.com/aces/Loris/pull/5533#discussion_r344882512